tools/migrate_pmid_to_uid/README.md

```diff
@@ -0,0 +1,46 @@
+## migrate_pmid_to_uid
```

Comment:
do you think we should add the reason or link to the details as to why we are doing this? Or is it a case of once its done it no longer matters?